### PR TITLE
adding implementation for validating login

### PR DESCRIPTION
**Pull request by Transurgeon**
adds implementation for ConcordiaID verification and email verification.

relevant issues: #4 and some progress towards #12.

### CODE REVIEW

**Comment by cedparadis** [comment]
of course your implementation does not work because you need to use your methods in RegisterSrv which process the data from the form. 
* Add implementation of your method into the RegisterSrv

**Review by cedparadis — DISMISSED** [review]
of course your implementation does not work because you need to use your methods in RegisterSrv which process the data from the form.

* Add implementation of your method into the RegisterSrv

**Comment by Transurgeon** [comment]
> of course your implementation does not work because you need to use your methods in RegisterSrv which process the data from the form.
> 
> * Add implementation of your method into the RegisterSrv

I made some changes to ``registerStudentUser`` which is called by RegisterSrv.
My issue is that even If I pass a valid ConcordiaID, it will flag as being wrong.

**Comment by Transurgeon** [comment]
now I am getting an error with the Jakarta email.. what should do about that?
Can you check if this issue is also on your end @cedparadis.
image

**Comment by Transurgeon** [comment]
Ok I just verified and we are able to create new users with the forms. Also the error message do display properly.
image
I think we can merge this for now. I will look into the Jakarta email at a later time.

File: src/com/shashi/service/impl/UserServiceImpl.java [hunk 1]
```diff
@@ -22,22 +22,31 @@ public String registerStudentUser(String userName, Long mobileNo, String emailId
...
+		if (!isValidEmail) {
+			return "Your email is not a valid Concordia email";
+		}
 
 		boolean isRegtd = isRegistered(user.getEmail(), user.getConcordiaId());
 
 		if (isRegtd) {
-			status = "Email Id Already Registered!";
-			return status;
+			return "Email Id Already Registered!";
```
Comment (chloedaull) [inline src/com/shashi/service/impl/UserServiceImpl.java:47]: pourquoi le return status et le return string qui dit la meme chose?

**Review by chloedaull — COMMENTED** [review]
excellent work overall, just have a question about what the ``isRegtd`` if statement returns

**Review by chloedaull — APPROVED** [review]

**Pull request merged**